### ISSUE TITLE
warning generated when building

### ISSUE THREAD

**Issue by igalic** [Fushko/gammy#53]
```sh
% c++ --version
FreeBSD clang version 11.0.0 (git@github.com:llvm/llvm-project.git llvmorg-11.0.0-0-g176249bd673)
Target: x86_64-unknown-freebsd13.0
Thread model: posix
InstalledDir: /usr/bin
```

when running make, this warning is generated:

```
src/utils.cpp:78:21: warning: unsequenced modification and access to 't' [-Wunsequenced]
                return -c / 2 * ((--t) * (t - 2) - 1) + b;
                                  ^       ~
1 warning generated.
```

**Comment by Fushko** [comment]
Happens on g++ as well.

**Comment by stefanos82** [comment]
Confirmed; I came here to report it too.

**Comment by Fushko** [comment]
Fixed in the develop branch

**Comment by stefanos82** [comment]
Cool; when you merge repo with master, let me know so I can re-build gammy.

**Comment by Fushko** [comment]
@stefanos82 fixed in 0.9.61
